src/drv_oculus_rift/rift.c

```diff
@@ -139,24 +222,213 @@ static void handle_tracker_sensor_msg(rift_priv* priv, unsigned char* buffer, in
 		vec3f_from_rift_vec(s->samples[i].accel, &priv->raw_accel);
 		vec3f_from_rift_vec(s->samples[i].gyro, &priv->raw_gyro);
 
+#if 0
```

Comment:
LOGD? I know we have to step up our logging game, but I really do dislike "#if 0"s.

Reply:
It should just go - not much point keeping them now it works, and easy to recreate if ever needed.